packages/react-native/src/views/ChatRoomView/ChatRoomView.js

```diff
@@ -16,7 +16,7 @@ const ChatRoomView = () => {
 	const { RCInstance, ECOptions } = useRCContext();
 
 	const isUserAuthenticated = useUserStore((state) => state.isUserAuthenticated);
-	const setRoles = useUserStore((state) => state.setRoles);
+	const setMemberRoles = useChannelStore((state) => state.setMemberRoles);
```

Comment:
Can we rather have useMemberStore here ? It is more related to member right ?

Reply:
Yes, we can do this. Either way this would work